pkg/webui/components/navigation/side/item/index.js

```diff
@@ -106,6 +113,8 @@ export class SideNavigationItem extends React.PureComponent {
             isExpanded={isExpanded}
             isMinimized={isMinimized}
             children={children}
+            currentPathName={location.pathname}
+            onDropdownItemsClick={this.handleItemClick}
           />
         ) : (
           <LinkItem
```

Comment:
I think you'll have to apply the same blur on click logic to the LinkItem as well, otherwise the dropdown will stay visible when the user clicks on this LinkItem instead of the dropdown item.

Reply:
Was not able to get this working with `event.target`, but `document.activeElement` did the job. Let me know if this needs an update as well.